airflow/kubernetes/pod_generator.py

```diff
@@ -292,6 +299,15 @@ def __init__(  # pylint: disable=too-many-arguments,too-many-locals
         # Attach sidecar
         self.extract_xcom = extract_xcom
 
+    def qualify_image_url(self, image: str) -> str:
+        if not self.docker_repository:
+            return image
+
+        if image.startswith(self.docker_repository):
+            return image
+
+        return self.docker_repository + '/' + image
```

Comment:
Please add a explicit image tag variable since the implicit tag 'latest' might not exist